prog/vnet/load_balancer_nexus.rb

```diff
@@ -104,6 +104,10 @@ def need_to_rewrite_dns_records?
     reap
     if strand.children.select { _1.prog == "Vnet::CertServer" }.all? { _1.exitval == "certificate is reshared" } || strand.children.empty?
       decr_refresh_cert
+      load_balancer.certs.reject { _1.id == load_balancer.active_cert.id }.each do |cert|
```

Comment:
```suggestion
      load_balancer.certs_dataset.exclude(id: load_balancer.active_cert.id).all do |cert|
```

Sequel does not technically support queries inside a `Dataset#each` block.  It happens to work with the postgres adapter, unless you try using streaming, but it's best not to rely on that.

Reply:
does it crash?

Reply:
On the postgres adapter, it does not by default.  This is because if you aren't using streaming, libpq will buffer all results in memory before iterating.  Many other Sequel adapters use a streaming approach by default, in which case, it can raise an exception (possibly crash depending on how the driver is programmed).  I consider it a bit habit to rely on this quirk of the postgres adapter, though.

Reply:
Thanks for the help here :) I have updated the PR.

Reply:
@jeremyevans yeah, but does it crash if doing the wrong thing? curious if I would have learned it in a graceful manner if not for the fact that I've been in libpq world for so long that it didn't wind up mattering

Reply:
With the way the postgres adapter works with libpq, it should not crash.  However, if we used sequel_pg's streaming support, or tried to support another database adapter, it could crash (or at least raise an exception).

Sequel::Dataset#each's documentation has a general warning against making queries inside an each block: https://sequel.jeremyevans.net/rdoc/classes/Sequel/Dataset.html#method-i-each